### PR TITLE
updated container orchestrator api-docs

### PR DESCRIPTION
**Pull request by GregoryIvo**
Release 5.1.0 brings many different changes to the container orchestrator. 

Brief description of the PR. [e.g. Added `null` check on `object` to avoid `NullPointerException`]

**Related Issue:** This PR fixes/closes {issue number}

**Description of the solution adopted:** A more detailed description of the changes made to solve/close one or more issues. If the PR is simple and easy to understand this section can be skipped.

**Screenshots:** If applicable, add screenshots to help explain your solution

**Any side note on the changes made:** Description of any other change that has been made, which is not directly linked to the issue resolution [e.g. Code clean up/Sonar issue resolution]

### CODE REVIEW

**Comment by vinayak19th** [comment]
Very helpful

File: _posts/builtin/2022-02-23-container-orchestration-provider.md [hunk 1]
```diff
@@ -14,52 +14,66 @@ The Container Orchestration service is composed of two bundles, one that exposes
...
 
-## Configuring the container
+##Configuring the container
 
 To begin configuring the container, look under **Services** and select the item which has the name set in the previous step.
 
 Containers may be configured using the following fields:
 
-- **Enabled** - When true, api will create the described container. When false api will not create the container, or will destroy the container if already running.
+- **Enabled** - When true, API will create the described container. When false the API will not create the container or will destroy the container if already running.
```
Comment (MMaiero) [inline _posts/builtin/2022-02-23-container-orchestration-provider.md]: > When true, API will create the described container.

When true, the service will create the defined container

Comment (MMaiero) [reply _posts/builtin/2022-02-23-container-orchestration-provider.md]: Maybe better like this?

File: _posts/builtin/2022-02-23-container-orchestration-provider.md [hunk 2]
```diff
@@ -14,52 +14,66 @@ The Container Orchestration service is composed of two bundles, one that exposes
...
   
-- **Image Name** - Describes the docker image that will be used to create the container. Remember to ensure that the selected image supports the architecture of the host machine, or else the container will not be able to start.
+- **Image Name** - Describes the image that will be used to create the container. Remember to ensure that the selected image supports the architecture of the host machine, or else the container will not be able to start.
   
-- **Image Tag** - Describes the version of the docker image that will be used to create the container.
+- **Image Tag** - Describes the version of the image that will be used to create the container.
+
+- **Authentication Registry URL** - URL for an alternative registry to pull images from. (If the field is left blank, credentials will be applied to Docker-Hub). Please see the [Authenticated Containers ](#authenticated-containers) document for more information about connecting to different popular repositories.
```
Comment (MMaiero) [inline _posts/builtin/2022-02-23-container-orchestration-provider.md]: > Please see the [Authenticated Containers ](#authenticated-containers) document

Please see the [Authenticated Containers ](#authenticated-containers) section

Comment (MMaiero) [reply _posts/builtin/2022-02-23-container-orchestration-provider.md]: > repositories

I believe that registries is more correct

File: _posts/builtin/2022-02-23-container-orchestration-provider.md [hunk 3]
```diff
@@ -14,52 +14,66 @@ The Container Orchestration service is composed of two bundles, one that exposes
...
+
+- **Authentication Username** - Describes the username to access the repository entered above.
```
Comment (MMaiero) [inline _posts/builtin/2022-02-23-container-orchestration-provider.md]: > repository

Same here

File: _posts/builtin/2022-02-23-container-orchestration-provider.md [hunk 4]
```diff
@@ -14,52 +14,66 @@ The Container Orchestration service is composed of two bundles, one that exposes
...
+
+- **Password** - Describes the password to access the alternative repository.
```
Comment (MMaiero) [inline _posts/builtin/2022-02-23-container-orchestration-provider.md]: > repository

Same here

File: _posts/builtin/2022-02-23-container-orchestration-provider.md [hunk 5]
```diff
@@ -14,52 +14,66 @@ The Container Orchestration service is composed of two bundles, one that exposes
...
 
-- **Logger Type** - This field provides a dropdown selection of supported container logging drivers.
-  
-- **Logger Parameters (optional)** - This field accepts a comma separated list of logging parameters. More information can be found in the [Docker logger documentation](https://docs.docker.com/config/containers/logging/configure/#supported-logging-drivers).
-  
+- **Logger Type** - This field provides a drop-down selection of supported container logging drivers.
+
+- **Logger Parameters (optional)** - This field accepts a comma-separated list of logging parameters.
+
+After specifying container parameters, ensure to set **Enabled** to **true** and press **Apply**. The container engine will then pull the respective image, spin up and start the container. If the gateway is reset (power cycled), and the container and Container Orchestration Service are set to **enabled**, The framework will automatically start the container again upon startup.
```
Comment (MMaiero) [inline _posts/builtin/2022-02-23-container-orchestration-provider.md]: > If the gateway is reset (power cycled), and the container and Container Orchestration Service are set to **enabled**, The framework will automatically start the container again upon startup.

If the gateway or the framework are power cycled, and the container and Container Orchestration Service are set to **enabled**, the framework will automatically start the container again upon startup.

File: _posts/builtin/2022-02-23-container-orchestration-provider.md [hunk 6]
```diff
@@ -71,91 +85,90 @@ To stop the container without deleting the component, set the **Enabled** field
...
 ## Container Management Dashboard
 
-The Container Orchestration service also provides the user with an intuitive container dashboard. This dashboard shows all containers running on a gateway, including containers created with Kura and those created manually through the command line interface. To utilize this dashboard the **org.eclipse.container.orchestration.provider** (Docker-API) must be enabled, and the dashboard can be opened by navigating to **Device > Docker Containers**.
+The Container Orchestration service also provides the user with an intuitive container dashboard. This dashboard shows all containers running on a gateway, including containers created with Kura and those created manually through the command line interface. To utilize this dashboard the **org.eclipse.container.orchestration.provider** (Container Orchestration Service) must be enabled, and the dashboard can be opened by navigating to **Device > Docker Containers**.
 
 ![Stop-Container]({{ site.baseurl }}/assets/images/builtin/container_orchestrator/container_inventory.png)
 
 ***
 
+# Authenticated Containers
```
Comment (MMaiero) [inline _posts/builtin/2022-02-23-container-orchestration-provider.md]: > Authenticated Containers

Maybe better `Authenticated Registries`?

Comment (GregoryIvo) [reply _posts/builtin/2022-02-23-container-orchestration-provider.md]: Agreed.

File: _posts/builtin/2022-02-23-container-orchestration-provider.md (hunk 1 above)
Comment (GregoryIvo) [reply _posts/builtin/2022-02-23-container-orchestration-provider.md]: Agreed!

**Pull request merged**